### ISSUE TITLE
sample generated is noise

### ISSUE THREAD

**Issue by guozixunnicolas** [deepsound-project/samplernn-pytorch#29]
Hi all, 

I have been training the model using google colab.
However, the generated sample is noise all the time. 
Please give me some suggestion;)

Best regards,

Zixun

**Comment by zguo008** [comment]
Issue resolved.
dataset is not labeled properly.

**Comment by Stafla** [comment]
Hi,

My generated samples tend to be noisy as well. The model I constructed has a final loss of  0.713 after 12 epochs. Should a model with a loss this low be able to generate music-resembling audio? 

@zguo008 could you please provide some more explanation about how you solved the issue? What do you exactly mean by "dataset is not labeled properly."? 

Kind regards, 

Sinit

**Comment by ghost** [comment]
@zguo008 were you able to train the model to not get random noise, also using google colab would appreciate a link to your google colab thanks
